demos/interactive_face_detection_demo_gapi/interactive_face_detection_gapi.hpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <gflags/gflags.h>
+
+static const char help_message[] =                   "Print a usage message";
+static const char input_video_message[] =            "Required. Path to a video file (specify \"cam\" to work with camera).";
+static const char output_video_message[] =           "Optional. Path to an output video file.";
+static const char face_detection_model_message[] =   "Required. Path to an .xml file with a trained Face Detection model.";
+static const char age_gender_model_message[] =       "Optional. Path to an .xml file with a trained Age/Gender Recognition model.";
+static const char head_pose_model_message[] =        "Optional. Path to an .xml file with a trained Head Pose Estimation model.";
+static const char emotions_model_message[] =         "Optional. Path to an .xml file with a trained Emotions Recognition model.";
+static const char facial_landmarks_model_message[] = "Optional. Path to an .xml file with a trained Facial Landmarks Estimation model.";
+static const char target_device_message[] =          "Optional. Target device for Face Detection network (the list of available devices is shown below). " \
```

Comment:
Aligning all help messages is a bad idea - if an option with a long name is added, we'll have to realign all of the messages. Just align the lines of each option separately. And remove the backslashes.

Reply:
I got you 
Done

Reply:
You didn't remove the backslashes, though...

Reply:
Backslashes removed